package.json

```diff
@@ -20,7 +20,7 @@
     "@types/color-hash": "1.0.2",
     "bootstrap": "3.4.1",
     "buffer": "6.0.3",
-    "chart.js": "2.9.4",
+    "chart.js": "3.8.2",
```

Comment:
Switching to 3.8.2 instead of the latest 3.9.1, as the latter has a bug related to rollup that causes type declarations to not be found, which makes type checks fail.